app/server.go

```diff
@@ -1242,6 +1304,10 @@ func doCheckNumberOfActiveUsersWarnMetricStatus(a *App) {
 	}
 }
 
+func hasBeenAckedOrShown(metricStatusFromStore string, warnMetric model.WarnMetric) bool {
+	return (metricStatusFromStore == model.WARN_METRIC_STATUS_ACK || (warnMetric.IsBotOnly && metricStatusFromStore == model.WARN_METRIC_STATUS_RUNONCE))
```

Comment:
```suggestion
	return metricStatusFromStore == model.WARN_METRIC_STATUS_ACK || (warnMetric.IsBotOnly && metricStatusFromStore == model.WARN_METRIC_STATUS_RUNONCE)
```

Reply:
I'll rearrange the parenthesis to be consistent with the observation above wrt to readability.